st2client/st2client/commands/keyvalue.py

```diff
@@ -188,10 +188,14 @@ def run(self, args, **kwargs):
         instance = KeyValuePair()
         instance.id = args.name  # TODO: refactor and get rid of id
         instance.name = args.name
-        instance.value = args.value
         instance.scope = args.scope
         instance.user = args.user
 
+        if not args.value:
+            instance.value = input("Please insert value for key: ")
```

Comment:
There is also ``getpass.getpass()`` which displays entered value masked on the screen.

But if we go with that approach, we will likely need to ask for confirmation (aka input the secret value twice).

Reply:
In a future PR, I think we could add `getpass.getpass()` support, but only turn it on if encrypt=True. I think this PR is good enough to merge as is.